lib/ansible/runner/credential/vmware_credential.rb

```diff
@@ -0,0 +1,23 @@
+module Ansible
+  class Runner
+    class VmwareCredential < Credential
+      def self.auth_type
+        "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::VmwareCredential"
+      end
+
+      # Modeled off of vmware injectors for awx:
+      #
+      #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/injectors.py#L63-L67
+      #
+      # TODO:  NickL:  Should we add support for the VMWARE_VALIDATE_CERTS var?
+      #
+      def env_vars
+        {
+          "VMWARE_USER"     => auth.userid || "",
+          "VMWARE_PASSWORD" => auth.password || "",
+          "VMWARE_HOST"     => auth.host || ""
+        }
```

Comment:
Delete blanks?

Reply:
Again, no since `awx` defaults all of these values:

https://github.com/ansible/awx/blob/1242ee2b6584f32cbc4267145ee0c0c6bc845049/awx/main/models/credential/injectors.py#L64-L66